easybuild/easyconfigs/c/CubeGUI/CubeGUI-4.8-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,64 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2019 Juelich Supercomputing Centre, Germany
+# Authors::   Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+easyblock = 'EB_Score_minus_P'
+
+name = 'CubeGUI'
+version = '4.8'
+
+homepage = 'https://www.scalasca.org/software/cube-4.x/download.html'
+description = """
+ Cube, which is used as performance report explorer for Scalasca and Score-P,
+ is a generic tool for displaying a multi-dimensional performance space
+ consisting of the dimensions (i) performance metric, (ii) call path, and
+ (iii) system resource. Each dimension can be represented as a tree, where
+ non-leaf nodes of the tree can be collapsed or expanded to achieve the
+ desired level of granularity.
+
+ This module provides the Cube graphical report explorer.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://apps.fz-juelich.de/scalasca/releases/cube/%(version)s/dist']
+sources = [SOURCELOWER_TAR_GZ]
+patches = [
+    ('CubeGUI-4.8_missing_compile_file.patch', 1),
+]
+checksums = [
+    '1df8fcaea95323e7eaf0cc010784a41243532c2123a27ce93cb7e3241557ff76',  # cubegui-4.8.tar.gz
+    '487697669f6a9a3455fabfdb65d720699070986776c465830ad417942e25e2dc',  # CubeGUI-4.8_missing_compile_file.patch
+]
+
+preconfigopts = 'autoreconf && autoreconf build-frontend/ && '
+
+builddependencies = [
+    # use same binutils version that was used when building GCCcore
+    ('binutils', '2.38'),
+    ('Autotools', '20220317'),
+    ('CubeLib', '4.8'),
```

Comment:
Please move CubeLib to `dependencies`.  In theory the libraries are properly rpath-ed, but they are required at runtime.  Thus, from an EB perspective, they are more than just `builddependencies`.

Reply:
Thanks and done, this one was migrated from previous easyconfigs.